include/senderCadmium.hpp

```diff
@@ -45,26 +45,26 @@ using namespace std;
         using defs=Sender_defs; // putting definitions in context
         public:
             //Parameters to be overwriten when instantiating the atomic model
-            TIME   preparationTime;
+            TIME   preparation_time;
```

Comment:
This is a constant and should be upper case.

Reply:
Changes done. Constants are in Upper case now. Thank You.